internal/controller/cephfscg/cephfscg_suite_test.go

```diff
@@ -115,7 +115,7 @@ var _ = BeforeSuite(func() {
 	k8sManager, err := ctrl.NewManager(cfg, ctrl.Options{
 		Scheme: scheme.Scheme,
 		Metrics: metricsserver.Options{
-			BindAddress: ":9090",
+			BindAddress: ":9289",
```

Comment:
In all other tests we set this to `0` to disable metrics, maybe we should do the same here?

Reply:
Port 0 gets a random port from the kernel that will never conflict with anything. If we want to use the service we need to discover the random port and configure to system to use it.

Reply:
setting to `0` disables metrics server, here is the [doc](https://github.com/kubernetes-sigs/controller-runtime/blob/main/pkg/metrics/server/server.go#L69)